README.md

```diff
@@ -296,12 +296,14 @@ Here are some examples of conditions you can use:
 
 
 ### Storage
-| Parameter         | Description                                                                                                                                        | Default    |
-|:------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|:-----------|
-| `storage`         | Storage configuration                                                                                                                              | `{}`       |
-| `storage.path`    | Path to persist the data in. Only supported for types `sqlite` and `postgres`.                                                                     | `""`       |
-| `storage.type`    | Type of storage. Valid types: `memory`, `sqlite`, `postgres`.                                                                                      | `"memory"` |
-| `storage.caching` | Whether to use write-through caching. Improves loading time for large dashboards. <br />Only supported if `storage.type` is `sqlite` or `postgres` | `false`    |
+| Parameter                        | Description                                                                                                                                        | Default    |
+|:---------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|:-----------|
+| `storage`                        | Storage configuration                                                                                                                              | `{}`       |
+| `storage.path`                   | Path to persist the data in. Only supported for types `sqlite` and `postgres`.                                                                     | `""`       |
+| `storage.type`                   | Type of storage. Valid types: `memory`, `sqlite`, `postgres`.                                                                                      | `"memory"` |
+| `storage.caching`                | Whether to use write-through caching. Improves loading time for large dashboards. <br />Only supported if `storage.type` is `sqlite` or `postgres` | `false`    |
+| `storage.maximumNumberOfResults` | The maximum number of results that an endpoint can have                                                                                            | `100`      |
+| `storage.maximumNumberOfEvents`  | The maximum number of events that an endpoint can have                                                                                             | `50`       |
```

Comment:
For consistency, `maximumNumberOfResults` and `maximumNumberOfEvents` should be `maximum-number-of-results` and `maximum-number-of-events` respectively.